process/throttle/antiflood/floodPreventers/quotaFloodPreventer.go

```diff
@@ -159,8 +211,8 @@ func (qfp *quotaFloodPreventer) increaseLoad(pid core.PeerID, size uint64) error
 	return nil
 }
 
-func (qfp *quotaFloodPreventer) isMaximumReached(absoluteMax uint64, counted uint64) bool {
-	max := uint64(100-qfp.percentReserved) * absoluteMax / 100
+func (qfp *quotaFloodPreventer) isMaximumReached(absoluteMax uint64, counted uint64, percentReserved float32) bool {
+	max := uint64(100-percentReserved) * absoluteMax / 100
```

Comment:
it is faster if you do multiplication instead of division. may delete / 100 and change return to counted * 100 > max - is it ensured that there are no overflows ? :)